common/importer/brave_importer_utils.cc

```diff
@@ -23,13 +23,17 @@ bool BraveImporterCanImport(const base::FilePath& profile,
     profile.Append(base::FilePath::StringType(FILE_PATH_LITERAL("Login Data")));
   base::FilePath cookies =
     profile.Append(base::FilePath::StringType(FILE_PATH_LITERAL("Cookies")));
+  base::FilePath ledger_state =
+    profile.Append(base::FilePath::StringType(FILE_PATH_LITERAL("ledger-state.json")));
 
   if (base::PathExists(session_store))
     *services_supported |= importer::HISTORY | importer::FAVORITES | importer::STATS;
   if (base::PathExists(passwords))
     *services_supported |= importer::PASSWORDS;
   if (base::PathExists(cookies))
     *services_supported |= importer::COOKIES;
+  if (base::PathExists(ledger_state))
```

Comment:
Minor, but: we should only add ledger import to the set of services_supported if *both* `ledger-state.json` and `session-store-1` are found, because we import data from both sources in `BraveImporter::ImportLedger`.